[2025-09-12T07:48:16.213+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: aviationstack_kafka_dag.run_kafka_consumer manual__2025-09-12T07:48:09.966613+00:00 [queued]>
[2025-09-12T07:48:16.216+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: aviationstack_kafka_dag.run_kafka_consumer manual__2025-09-12T07:48:09.966613+00:00 [queued]>
[2025-09-12T07:48:16.216+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-12T07:48:16.220+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_kafka_consumer> on 2025-09-12 07:48:09.966613+00:00
[2025-09-12T07:48:16.221+0000] {standard_task_runner.py:60} INFO - Started process 1353 to run task
[2025-09-12T07:48:16.223+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'aviationstack_kafka_dag', 'run_kafka_consumer', 'manual__2025-09-12T07:48:09.966613+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/aviationstack_dag.py', '--cfg-path', '/tmp/tmp9giz4gsi']
[2025-09-12T07:48:16.224+0000] {standard_task_runner.py:88} INFO - Job 17: Subtask run_kafka_consumer
[2025-09-12T07:48:16.246+0000] {task_command.py:423} INFO - Running <TaskInstance: aviationstack_kafka_dag.run_kafka_consumer manual__2025-09-12T07:48:09.966613+00:00 [running]> on host eba0e74bcf84
[2025-09-12T07:48:16.274+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='aviationstack_kafka_dag' AIRFLOW_CTX_TASK_ID='run_kafka_consumer' AIRFLOW_CTX_EXECUTION_DATE='2025-09-12T07:48:09.966613+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-12T07:48:09.966613+00:00'
[2025-09-12T07:48:16.275+0000] {kafka_consumer.py:54} INFO - Starting Kafka consumer...
[2025-09-12T07:48:16.277+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.4', 9092)]>: connecting to kafka:9092 [('172.18.0.4', 9092) IPv4]
[2025-09-12T07:48:16.277+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>: Connection complete.
[2025-09-12T07:48:16.277+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>: Closing connection. 
[2025-09-12T07:48:16.382+0000] {cluster.py:401} INFO - Coordinator for group/aviation_flight_data_test_group_01 is ('coordinator-1', 'kafka', 9092, None)
[2025-09-12T07:48:16.382+0000] {base.py:788} INFO - Discovered coordinator coordinator-1 for group aviation_flight_data_test_group_01
[2025-09-12T07:48:16.382+0000] {base.py:848} INFO - Starting new heartbeat thread
[2025-09-12T07:48:16.383+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group aviation_flight_data_test_group_01
[2025-09-12T07:48:16.383+0000] {base.py:419} INFO - Failed to join group aviation_flight_data_test_group_01: NodeNotReadyError: coordinator-1
[2025-09-12T07:48:16.384+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.4', 9092)]>: connecting to kafka:9092 [('172.18.0.4', 9092) IPv4]
[2025-09-12T07:48:16.384+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>: Connection complete.
[2025-09-12T07:48:16.486+0000] {base.py:532} INFO - (Re-)joining group aviation_flight_data_test_group_01
[2025-09-12T07:48:16.487+0000] {base.py:637} INFO - Received member id kafka-python-2.2.15-c8ad0451-691a-4589-80a0-e90777eef1a4 for group aviation_flight_data_test_group_01; will retry join-group
[2025-09-12T07:48:16.488+0000] {base.py:419} INFO - Failed to join group aviation_flight_data_test_group_01: [Error 79] MemberIdRequiredError
[2025-09-12T07:48:16.488+0000] {base.py:532} INFO - (Re-)joining group aviation_flight_data_test_group_01
[2025-09-12T07:48:19.497+0000] {base.py:594} INFO - Successfully joined group aviation_flight_data_test_group_01 <Generation 1 (member_id: kafka-python-2.2.15-c8ad0451-691a-4589-80a0-e90777eef1a4, protocol: range)>
[2025-09-12T07:48:19.497+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-09-12T07:48:19.502+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('aviation_flight_data', 0)]
[2025-09-12T07:48:19.502+0000] {consumer.py:249} INFO - Setting newly assigned partitions (('aviation_flight_data', 0),) for group aviation_flight_data_test_group_01
[2025-09-12T07:48:19.504+0000] {fetcher.py:418} INFO - Resetting offset for partition ('aviation_flight_data', 0) to offset 5.
[2025-09-12T07:48:26.275+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-12T07:48:26.282+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=aviationstack_kafka_dag, task_id=run_kafka_consumer, execution_date=20250912T074809, start_date=20250912T074816, end_date=20250912T074826
[2025-09-12T07:48:26.296+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-12T07:48:26.311+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
